### PR TITLE
Added Explore Page

### PR DESCRIPTION
**Pull request by sanjitsarkar**

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Clean and modular code
* Good work with redux toolkit
* Looks good

File: src/pages/HomePage/PostsSection.jsx [hunk 1]
```diff
@@ -2,16 +2,28 @@ import React, { useEffect } from "react";
...
+    if (type === "userFeed") {
+      dispatch(fetchUserFeedPosts());
+    } else if (type === "all") {
+      dispatch(fetchAllPosts());
+    }
+
   }, []);
   return (
-    <PostsWrapper posts={posts}>
+    <PostsWrapper posts={type === "all" ? allPosts : posts}>
```
Comment (deekshithmd) [inline src/pages/HomePage/PostsSection.jsx:26]: Not able to login/signup...please check this whether it is expected behaviour or need to be fixed, code looks clean and understandable...

**Review by deekshithmd — APPROVED** [review]

File: src/pages/HomePage/PostsSection.jsx (hunk 1 above)
Comment (sanjitsarkar) [reply src/pages/HomePage/PostsSection.jsx:26]: you have to visit [ZenGram](https://zengram.netlify.app/)

File: src/pages/index.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import LoginPage from "./AuthPage/LoginPage";
 import SignupPage from "./AuthPage/SignupPage";
 import BookmarkedPage from "./BookmarkedPage";
+import ExplorePage from "./ExplorePage";
 import HomePage from "./HomePage";
-export { HomePage, LoginPage, SignupPage, BookmarkedPage };
+export { HomePage, LoginPage, SignupPage, BookmarkedPage, ExplorePage };
```
Comment (jalajcodes) [inline src/pages/index.jsx:6]: you can also directly export on the same line.

`export { default as LoginPage } from "./AuthPage/LoginPage";`

**Review by jalajcodes — APPROVED** [review]
Code looks clean. Good work!

File: src/pages/index.jsx (hunk 2 above)
Comment (sanjitsarkar) [reply src/pages/index.jsx:6]: Learnt something new

**Pull request merged**